sdk/typescript/src/rpc/faucet-client.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import fetch from 'cross-fetch';
```

Comment:
As an aside, I'd like to maybe explore making users polyfill fetch rather than using a library here. Not 100% convinced just thinking out loud.

Reply:
This library only costs [2.8kb](https://bundlephobia.com/package/cross-fetch@3.1.5) and can save people time from debugging cross-platform bugs